content/recipes/context.mdx

```diff
@@ -204,7 +204,7 @@ In a complex app where stores might share dependencies, you might have to defer
 Finally, let's make our custom `useStores` hook to access the exported `storesContext` value.
 
 ```tsx
-// src/hooks/user-store.tsx
+// src/hooks/use-store.tsx
```

Comment:
When you are fixing, make sure it's correct fix ;)
```suggestion
// src/hooks/use-stores.tsx
```

Reply:
Thanks! I fixed it so please review :)